### PR TITLE
feat: add tooltip for original sale toggle

### PR DESCRIPTION
**Pull request by EsterB12**

### CODE REVIEW

File: components/nft-sale-widget/commons/sale-type-switch.tsx [hunk 1]
```diff
@@ -24,13 +25,25 @@ export const SaleTypeSwitch = ({
...
+
       <Text
         fontFamily={"Inter Medium"}
         fontSize={{ base: 12, md: 14 }}
         color={"#585858"}
       >
         Original Sale
       </Text>
+      <Tooltip
+        label="Toggle to view available copies"
```
Comment (saksijas) [inline components/nft-sale-widget/commons/sale-type-switch.tsx:37]: what if we are already seeing copies?

Comment (saksijas) [reply components/nft-sale-widget/commons/sale-type-switch.tsx:37]: Maybe this message should change depending on toggle state

Comment (EsterB12) [reply components/nft-sale-widget/commons/sale-type-switch.tsx:37]: Great idea! I will implement

**Review by saksijas — APPROVED** [review]

File: components/nft-sale-widget/commons/sale-type-switch.tsx [hunk 2]
```diff
@@ -24,13 +25,44 @@ export const SaleTypeSwitch = ({
...
       </Text>
+      <Tooltip
+        label={
+          isOriginal ? (
+            <Text>
+              Toggle to view{" "}
+              <Show breakpoint="(max-width: 480px)">
+                <br />
+              </Show>{" "}
+              NFT copies
```
Comment (saksijas) [inline components/nft-sale-widget/commons/sale-type-switch.tsx:44]: As you can see only thing to change is the L44. and L52. No need to do entire text only partial text

**Pull request merged**